tests/efi-vars-editor-vm

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+set -eux
+
+architecture="$(uname -m)"
+if [ "${architecture}" != "x86_64" ] && [ "${architecture}" != "aarch64" ]; then
+  echo "Skipping test as EFI is not supported on ${architecture}"
```

Comment:
This would need the following to stop here:
```
FAIL=0
exit 0
```

Is the `lxc config uefi` support not advertised via an API extension? Or is that advertised for all arches because API extensions are generic?

Reply:
It's advertised for all arches, yes, because API extensions are generic.

And yes, this is definitely a copy-n-paste mistake, because I was looking on `tests/cpu-vm` as an example. It has the same issue.